utils/kata-manager.sh

```diff
@@ -777,7 +822,25 @@ handle_installation()
 	local kata_version="${7:-}"
 	local containerd_flavour="${8:-}"
 
-	[ "$only_run_test" = "true" ] && test_installation && return 0
+	local install_docker="${9:-}"
+	[ -z "$install_docker" ] && die "no install docker value"
+
+	# The tool to be testing the installation with
+	local tool="ctr"
+
+	if [ "$install_docker" = "true" ]
+	then
+		if [ "$skip_containerd" = "false" ]
+		then
+			# The script provided by docker already takes care
+			# of properly installing containerd
+			skip_containerd="true"
+			info "Containerd will be instaslled during the Docker installation.\nThe '-c' option is being ignored."
```

Comment:
Nit: Typo fix + I'd keep this all on one line:

```suggestion
			info "Containerd will be installed during the Docker installation ('-c' option ignored)"
```